testing/wireguard-hardened/APKBUILD

```diff
@@ -10,7 +10,7 @@ _kpkgrel=0
 # we must also match up _toolsrel with wireguard-tools
 _ver=0.0.20180420
 _mypkgrel=2
```

Comment:
This is called `_rel` in the vanilla one. Maybe you want to rename this too, to keep it consistent, while you're at it?